config-connector/solutions/projects/kpt/project-services/service.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+kind: Service
+metadata:
+  annotations:
+    # Replace the ${PROJECT_ID?} below with your desired project ID.
+    cnrm.cloud.google.com/project-id: ${PROJECT_ID?} # {"$ref":"#/definitions/io.k8s.cli.setters.project-id"} 
```

Comment:
Let's follow the best practice here: https://github.com/maqiuyujoyce/cloud-foundation-toolkit/pull/2/files

Reply:
Will this mean that applying the yamls will be a two step process? i.e. first apply the namespace then apply the service with `kubectl apply -f service.yaml --namespace project-annotated`

Reply:
No, just `kubectl apply -f .` is sufficient. K8s will handle the reconciliation properly.